views/partials/footer.tmpl

```diff
@@ -1,2 +1,8 @@
 {{define "footer"}}
+	<footer class="container outer footer">
+        	<div class="navbar-left"><a href="#">Disclaimer</a> | <a href="#">MyRadio</a> | <a href="#">Contact</a></div>
+        	<div class="navbar-right">
+				<p>Copyright © 1967-(Insert Current Year Here) University Radio York. Designed by the <a href="mailto:webmaster@ury.org.uk" title="Click to email the Computing team.">URY Computing Team</a></p>
```

Comment:
A lot of places do actually update the copyright notices automatically (a quick look on a search engine of your choice will throw up a bunch of tutorials etc. on how to achieve this). However copyright notices are actually legally fairly meaningless, and you have copyright on a work by default, whether or not you state it. Including a start date can sometimes make copyright claims easier however, if you don't have other proof that your work came first.